File: linkedlist/ex.java
import java.util.*;
import java.io.*;
class Node{
    int data;
    Node next;
    Node(int x){
        data=x;
        next=null;
    }
}



class ex
{
    public static void main(String[] args) {
        Node head=new Node(20);
        head.next=new Node(20);
        head.next.next=new Node(30);
        printlist(head);
    }
    public static void printlist(Node head){
        while(head!=null){
            System.out.print(head.data+" ");
            head=head.next;
        }
    }
}

File: linkedlist/ex.cpp
#include<iostream>
using namespace std;
struct Node
{
    int data;
    Node *next;
    Node(int x){
        data=x;
        next=NULL;
    }
};

void printlist(Node *head){
    while(head!=NULL)
    {
        cout<<head->data<<" ";
        head=head->next;
    }
}

int main(){
    Node  *head=new Node(10);
    head->next=new Node(20);
    head->next->next=new Node(20);
    printlist(head);
    return 0;
}
